.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,35 +1,41 @@
 <!--
-    If you have any questions regarding your scenario or components behavior, please provide a detailed explanation.
-    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask) to receive an answer shortly.
+    We cannot guarantee a quick answer on GitHub.
+    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask)
+    to receive an answer shortly.
 -->
 
 <!--
-    Checked checkbox should look like this: [x]
+    A checked check box should look like this: [x]
 -->
 - [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.
 
 ### I'm using ...
-<!--
-    Checked checkbox should look like this: [x]
--->
+
 - [ ] React Grid
 - [ ] React Chart
 - [ ] React Scheduler
 
 ### Current Behaviour
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
```

Comment:
```suggestion
<!-- A detailed description of the problem. For example, I'm always frustrated when [...] -->
```

Reply:
Why 'I'm always frustrated when'?

Reply:
That's just an example of how you can start your description of the problem
```suggestion
<!-- A detailed description of the problem. For example, "I'm always frustrated when [...]" -->
```

Reply:
Ok